db-manager/data-import-manuscript-version.mmd

```diff
@@ -0,0 +1,9 @@
+sequenceDiagram
+  main ->> dim Manuscript Version: stage_csv
+  main ->> dim Manuscript Version: applyChanges
+  dim Manuscript Version ->> dim Manuscript Version: cascadeActivations
+  dim Manuscript Version ->> dim Manuscript Version: cascadeRetirements
+  dim Manuscript Version ->> dim Manuscript: applyChanges
+  dim Manuscript Version ->> dim Manuscript: _applyChanges
```

Comment:
is this line necessary?

Reply:
That was actually wrong. I corrected it, it's called on itself (I could document separately what happens in that method).